fastlane/lib/fastlane/actions/device_grid/device_grid.rb

```diff
@@ -0,0 +1,102 @@
+# rubocop:disable Style/IndentationConsistency
+# We disable this rule since we intend opening HTML labels
+module Danger
+  class Dangerfile
+    module DSL
+      # A danger plugin: https://github.com/danger/danger
+      class DeviceGrid < Plugin
```

Comment:
I still think this should have a few tests, at least something that makes sure the rendering isn't totally broken.